framework/wait/state/activeCluster.go

```diff
@@ -0,0 +1,44 @@
+package functions
+
+import (
+	"time"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	activeState   = "active"
+	errorState    = "error"
+	True          = "True"
+	waitingState  = "waiting"
+	updatingState = "updating"
+)
+
+// IsActiveCluster is a function that will wait for the cluster to be in an active state.
+func IsActiveCluster(client *rancher.Client, clusterID string) (done bool, err error) {
```

Comment:
nit for all of the functions that return a `done` bool; is that being used anywhere? I feel like it is not used at all for any of the functions that have them right now.

Reply:
Good point, they don't use them after double-checking. I removed them and left error there in case future functions we will need to provide some error checking.